README.rst

```diff
@@ -19,3 +19,12 @@ To use it, add ``sphinxcontrib.jquery`` as a Sphinx extension:
        "sphinxcontrib.jquery",
    ]
    ...
+
+Configuration
+-------------
+
+```
+# Enable CORS integrity="<hash>" on included JS files
```

Comment:
What is `CORS` integrity? Do you mean SRI?

Reply:
I was duped by Firefox to think that SRI was part of CORS: https://github.com/readthedocs/sphinx_rtd_theme/issues/1420#issuecomment-1402117821

It doesn't seem like it's directly related though, and it even seems that the SRI check should happen before CORS: https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity#how_browsers_handle_subresource_integrity

> the browser must first compare the script or stylesheet to the expected hash given in the integrity value.

Gathering that naming this as `jquery_cors_enable` won't work, so will rename.

Reply:
Renamed to `jquery_sri_enable`